.circleci/config.yml

```diff
@@ -195,7 +195,7 @@ jobs:
               mkdir graph-test-results
               neo4j start
               # poll for neo4j status until available
-              while ! neo4j status; do sleep 1; done
+              while ! curl --fail http://127.0.0.1:7474 &> /dev/null; do sleep 1; done
```

Comment:
Please add a log message on each iteration, so we know what's going on

Reply:
I can remove the `&> /dev/null` to print the original output, which is a connection error. Personally, I think it just spams the log and will make people think that there's an error even though everything works as expected. The connection should take at most 10 seconds.